packages/runner-ct/cypress/component/app.spec.tsx

```diff
@@ -0,0 +1,88 @@
+/// <reference types="@percy/cypress" />
```

Comment:
It looks like now it will have 2 files: App.spec and app.spec

Reply:
I pushed a fix changing this file to be `App.spec.tsx` to match the one in develop